certified-connectors/WithoutWire Inventory Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1204 @@
+                      }
+                    },
+                    "description": "LineItems"
+                  },
+                  "LoadDate": {
+                    "type": "string",
+                    "description": "LoadDate"
+                  },
+                  "PrintDate": {
+                    "type": "string",
+                    "description": "PrintDate"
+                  },
+                  "ExportDate": {
+                    "type": "string",
+                    "description": "ExportDate"
+                  },
+                  "ParentOrderNumber": {
+                    "type": "string",
+                    "description": "ParentOrderNumber"
+                  },
+                  "TrackingNumber": {
+                    "type": "string",
+                    "description": "TrackingNumber"
+                  },
+                  "ShippingMethodName": {
+                    "type": "string",
+                    "description": "ShippingMethodName"
+                  },
+                  "ShippingWeight": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "ShippingWeight"
+                  },
+                  "ShippingCost": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "ShippingCost"
+                  },
+                  "Status": {
+                    "type": "string",
+                    "description": "Status"
+                  },
+                  "StatusCode": {
+                    "type": "string",
+                    "description": "StatusCode"
+                  },
+                  "Identity": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "Identity"
+                  },
+                  "UseOrderLevelShipping": {
+                    "type": "string",
+                    "description": "UseOrderLevelShipping"
+                  },
+                  "OwnerName": {
+                    "type": "string",
+                    "description": "OwnerName"
+                  },
+                  "PickingPriority": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "PickingPriority"
+                  },
+                  "RoutePickingPriority": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "RoutePickingPriority"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get work orders",
+        "operationId": "GetWorkOrders",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "orderStatusCode",
+            "in": "query",
+            "required": false,
+            "type": "string",
+			"description": "Work order status (status code)",
```

Comment:
@PavloNikulinEnavate Please fix the alignments of all the properties in the swagger and apiProperties.